docs/testing.md

```diff
@@ -1,10 +1,8 @@
 # Testing 
 
-Running `swift test` from the terminal will trigger the unit tests (or triggering the test action in Xcode).
+SwiftPM is integrated with [XCTest, Apple’s unit test framework](https://developer.apple.com/documentation/xctest). Running `swift test` from the terminal, or triggering the test action in Xcode, will run all of your XCTest test cases. Test results will be displayed in Xcode or printed out to the terminal.
```

Comment:
Maybe we can be a bit less xcode specific and just say "IDE (like Xcode or similar)" and then continue using IDE?

Reply:
Love the intro tho, it is super unclear to newcomers what the hell xctest is and if they have to use it or not :)

Reply:
good point, done ✅